components/LoginForm.vue

```diff
@@ -78,57 +77,12 @@ export default Vue.extend({
     ...mapActions('user', ['setUser', 'clearUser']),
     ...mapActions('deviceProfile', ['setDeviceProfile']),
     ...mapActions('snackbar', ['pushSnackbarMessage']),
-    async userLogin() {
-      this.loginIn = true;
-      try {
-        this.$axios.setBaseURL(this.serverUrl);
-
-        const serverInfo = (await this.$api.system.getPublicSystemInfo()).data;
-
-        if (compareVersions.compare(serverInfo.Version || '', '10.7.0', '>=')) {
-          const response = await this.$auth.loginWith('local', {
-            data: this.login
-          });
-
-          this.setDeviceProfile();
-
-          const accessToken = `MediaBrowser Client="${this.$store.state.deviceProfile.clientName}", Device="${this.$store.state.deviceProfile.deviceName}", DeviceId="${this.$store.state.deviceProfile.deviceId}", Version="${this.$store.state.deviceProfile.clientVersion}", Token="${response.data.AccessToken}"`;
-
-          this.$auth.setUserToken(accessToken);
-
-          this.$auth.setUser(response.data.User);
-          this.setUser({
-            id: response.data.User.Id,
-            serverUrl: this.serverUrl,
-            accessToken
-          });
-        } else {
-          this.loginIn = false;
-          this.pushSnackbarMessage({
-            message: this.$t('serverVersionTooLow'),
-            color: 'error'
-          });
-        }
-      } catch (error) {
-        let errorMessage = this.$t('unexpectedError');
-
-        if (!error.response) {
-          errorMessage = this.$t('serverNotFound');
-        } else if (
-          error.response.status === 500 ||
-          error.response.status === 401
-        ) {
-          errorMessage = this.$t('incorrectUsernameOrPassword');
-        } else if (error.response.status === 400) {
-          errorMessage = this.$t('badRequest');
-        }
-
-        this.loginIn = false;
-        this.pushSnackbarMessage({
-          message: errorMessage.toString(),
-          color: 'error'
-        });
-      }
+    userLogin() {
```

Comment:
Why no longer async? It's because ``this.login`` is async now right?

Reply:
Nuxt Auth handles everything internally. You just need to call it normally. So the Nuxt Auth "loginWith" isn't async :)